### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by heatherbooker**
nav bar component created using react , with (newly organized) scss for styling

### CODE REVIEW

**Comment by heatherbooker** [comment]
@dharness CIGAR?

**Comment by dharness** [comment]
@heatherbooker No thanks

**Comment by dharness** [comment]
@heatherbooker please remove the prototype folder in a seperate PR, it makes reviewing much more difficult

**Comment by heatherbooker** [comment]
@dharness done

File: src/index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
 	<title>Frenchly</title>
+  <link href='https://fonts.googleapis.com/css?family=Raleway:300,500,600' rel='stylesheet' type='text/css'>
```
Comment (dharness) [inline src/index.html:5]: Please use consistent indentation

File: src/index.jsx [hunk 2]
```diff
@@ -29,6 +43,6 @@ var Badge = React.createClass({
 
 //render react component
 ReactDOM.render(
-  <Main />,
+  <MainDiv />,
```
Comment (dharness) [inline src/index.jsx:46]: Why did you change this?

File: src/stylesheets/main.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
+$main-blue: #20A6e7;
```
Comment (dharness) [inline src/stylesheets/main.scss]: Please move these vars into a separate variables.scss file

File: src/stylesheets/main.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+$font-lite: 400;
+$font-med: 500;
+$font-bold: 600;
+
+//basic browser reset styles
+@import "partials/reset";
```
Comment (dharness) [inline src/stylesheets/main.scss:2]: @imports at the top please

File: src/index.jsx (hunk 2 above)
Comment (heatherbooker) [inline src/index.jsx:46]: I found it ambigious to just call it Main...is there a standard i should follow?

File: src/index.html (hunk 1 above)
Comment (heatherbooker) [inline src/index.html:5]: how it looks in my IDE: http://puu.sh/pfXEt/b3706f282b.png

the problem: 2 spaces vs a tab

**Comment by heatherbooker** [comment]
@dharness done
